src/themes/OLH/templates/journal/authors.html

```diff
@@ -0,0 +1,43 @@
+{% extends "core/base.html" %}
+{% load static from staticfiles %}
+
+{% block title %}Authors{% endblock title %}
+{% block page_title %}Authors{% endblock page_title %}
+
+{% block body %}
+
+    <h3>Authors</h3>
+    <div class="row" data-equalizer data-equalize-on="medium">
+       
+        {% for author in author_list %}
+            <div class="medium-3 columns {% if forloop.last %}end{% endif %}">
+                <div class="{% if journal_settings.general.enable_editorial_images %}editorial-team{% endif %} callout" data-equalizer-watch>
+                   
+                    {% if journal_settings.general.enable_editorial_images %}
+                        <img class="thumbnail editorial-image" src="{% if author.profile_image %}{{ author.profile_image.url }}{% else %}{% static "common/img/icons/users.png" %}{% endif %}"
```

Comment:
Could use `default_if_none` here: https://docs.djangoproject.com/en/1.11/ref/templates/builtins/#default-if-none
Although I think you are trying to be consistent with other templates

Reply:
Yeah I was trying to be consistent.
on
`{{ author.profile_image.url|default_if_none:'just/to/test.png' }}`
I get a valueerror when updating to
`The 'profile_image' attribute has no file associated with it.`
should I just stick with what's currently there?

Reply:
Ah fair enough, the lack of a `profile_image` is the problem, not the lack of a `url`. Yes, leave it as it is